experiments/interpretation/dutchf3_patch/local/test.py

```diff
@@ -267,28 +267,42 @@ def _evaluate_split(
 
     n_classes = test_set.n_classes
 
+    if debug:
+        data_flow[split] = dict()
+        data_flow[split]['test_section_loader_length'] = len(test_set)
+        data_flow[split]['test_input_shape'] = test_set.seismic.shape
+        data_flow[split]['test_label_shape'] = test_set.labels.shape
+        data_flow[split]['n_classes'] = n_classes
+
+
     test_loader = data.DataLoader(test_set, batch_size=1, num_workers=config.WORKERS, shuffle=False)
 
     if debug:
+        data_flow[split]['test_loader_length'] = len(test_loader)
         logger.info("Running in Debug/Test mode")
-        test_loader = take(2, test_loader)
+        take_n = 2
+        test_loader = take(take_n, test_loader)
+        data_flow[split]['take_n_sections'] = take_n
```

Comment:
so we're testing that there are 2 sections available for scoring, is this correct?

Reply:
I use this to test the length of predictions, labels, and images coming from model evaluation!